lib/releaselib.sh

```diff
@@ -190,7 +190,7 @@ release::set_build_version () {
         logecho "$ATTENTION: The $branch branch HEAD is ahead of the last" \
                 "good Jenkins run by $commit_count commits." \
                 "Wait for Jenkins to catch up."
-        return 1
+        ((FLAGS_allow_stale_build)) || return 1
```

Comment:
```
If ((FLAGS_allow_stale_build)); then
  logecho
else
  return 1
fi
```

Same with the other.  This will provide a more visually spaced output for those messages between the job lists.